addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1549,13 +1549,15 @@ const Wysiwyg = Widget.extend({
                 return;
             }
             $image.transfo({document: this.odooEditor.document});
+            imgTransformBtn.setAttribute('title', 'Click again to reset transformation');
             const mouseup = () => {
                 imgTransformBtn.classList.toggle('active', $image[0].matches('[style*="transform"]'));
             };
             $(this.odooEditor.document).on('mouseup', mouseup);
             const mousedown = mousedownEvent => {
                 if (!$(mousedownEvent.target).closest('.transfo-container').length) {
                     $image.transfo('destroy');
+                    imgTransformBtn.setAttribute('title', 'Transform the picture (click twice to reset transformation)');
```

Comment:
Those strings won't be translated

Reply:
@dmo-odoo Thanks for noticing, I've updated the code!